### ISSUE TITLE
Possible false positive for comments within a run command.

### ISSUE THREAD

**Issue by deepio** [AtomLinter/linter-docker#56]
I'm getting a `Empty / bogus instruction is invalid`

This is part of my dockerfile. This runs correctly in docker. 

```
RUN apk update \
  # psycopg2 dependencies
  && apk add --virtual build-deps gcc python3-dev musl-dev \
  && apk add postgresql-dev \
```

I have multiple lines after this one that start with `&&` and they do not have this error.

**Comment by Arcanemagus** [comment]
You'll want to file this over on [dockerlint](https://github.com/RedCoolBeans/dockerlint), this is just a wrapper around that 😉.
